src/test/regress/expected/distributed_types.out

```diff
@@ -380,6 +380,39 @@ SELECT run_command_on_workers($$SELECT count(*) FROM pg_type where typname IN ('
 (2 rows)
 
 RESET citus.enable_create_type_propagation;
+CREATE TYPE ct1 as (int_1 int, int_2 int);
+CREATE TABLE field_indirection_test_1 (int_col int, ct1_col ct1);
+SELECT create_distributed_table('field_indirection_test_1', 'int_col');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+INSERT INTO field_indirection_test_1 (int_col, ct1_col.int_1, ct1_col.int_2) VALUES (0, 1, 2);
+SELECT * FROM field_indirection_test_1 ORDER BY int_col, ct1_col;
+ int_col | ct1_col
+---------------------------------------------------------------------
+       0 | (1,2)
+(1 row)
+
+CREATE TYPE ct2 as (int_2 int, text_1 text, int_1 int);
+CREATE TABLE field_indirection_test_2 (int_col int, ct2_col ct2, ct1_col ct1);
+SELECT create_distributed_table('field_indirection_test_2', 'int_col');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+INSERT INTO field_indirection_test_2 (ct2_col.int_1, int_col, ct2_col.text_1, ct1_col.int_2) VALUES (0, 1, 'text1', 2);
+INSERT INTO field_indirection_test_2 (ct2_col.int_1, int_col, ct2_col.text_1, ct1_col.int_2) VALUES (3, 4, 'text1', 5);
```

Comment:
If we do it in a single command, i.e as a multi-row insert, it fails, looking into that now

Reply:
```sql
[local] postgres@postgres:9700-14342=# INSERT INTO field_indirection_test_2 (ct2_col.int_1, int_col, ct2_col.text_1, ct1_col.int_2) VALUES (0, 1, 'text1', 2), (3, 4, 'text1', 5);
NOTICE:  issuing BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;SELECT assign_distributed_transaction_id(0, 8, '2020-10-12 16:52:49.86752+03');
DETAIL:  on server postgres@localhost:9702 connectionId: 4
NOTICE:  issuing INSERT INTO public.field_indirection_test_2_102009 AS citus_table_alias (int_col, ct2_col, ct1_col) VALUES (1,ROW("*VALUES*".column1, "*VALUES*".column3),"*VALUES*".column4)
DETAIL:  on server postgres@localhost:9702 connectionId: 4
NOTICE:  issuing ROLLBACK
DETAIL:  on server postgres@localhost:9702 connectionId: 4
ERROR:  missing FROM-clause entry for table "*VALUES*"
```

Reply:
This is not introduced in this pr but I think it should still be fixed in this pr.